advanced/helm/is-pattern-2/values.yaml

```diff
@@ -33,9 +40,15 @@ wso2:
         initialDelaySeconds: 250
         periodSeconds: 10
       resources:
+        # These are the minimum resource recommendations for running WSO2 Enterprise Integrator server profiles
```

Comment:
@Savidude these comments have to change such that they are applicable to WSO2 Identity Server.

Reply:
Fixed https://github.com/wso2/kubernetes-is/pull/159/commits/e5afef47cc931b20789608fccfb3be0b0513fa3f